x/stakeibc/client/cli/gov.go

```diff
@@ -82,9 +82,6 @@ Where proposal.json contains:
 			if err != nil {
 				return err
 			}
-			if err := msg.ValidateBasic(); err != nil {
-				return err
-			}
```

Comment:
wdym these are now optional for gov messages?

Reply:
ValidateBasic is now optional for msgs that doesn't need it, apparently

Reply:
there's a way of checking it if it's available but in this instance we know that message doesn't have ValidateBasic() anymore so there's no sense in checking if ValidateBasic() is available

Reply:
example: https://github.com/Stride-Labs/osmosis/blob/2ce269b07/osmoutils/osmocli/cli_tester.go#L64-L66

Reply:
I think we do have validate basic for these though ([example](https://github.com/Stride-Labs/stride/blob/2dcced3b9260d2b0197cc41c6626efd264b536f0/x/stakeibc/types/gov.go#L46-L66))

Although, I think validateBasic is actually called in `GenerateOrBroadcastTxCLI` so no need to be redundant here

Reply:
x/gov's MsgSubmitProposal doesn't have ValidateBasic() anymore. In this instance the internal msg can have ValidateBasic() but what was removed here was the call to the outer MsgSubmitProposal ValidateBasic(). We can maybe add a call to `proposal.ValidateBasic()`.

Reply:
wdyt about da4b5bdba631a6b588545cd0562262b989d7adce?

Reply:
perfect